Tests/Functional/Sources/ApplicationStateHandlingTest.m

```diff
@@ -78,6 +78,25 @@ - (void)testOpenSettingsApplicationAndReturning_flaky {
       assertWithMatcher:grey_sufficientlyVisible()];
 }
 
+- (void)testApplicationRestartOnce {
```

Comment:
mind adding comment on top of these explaining what they do? something like:

/** Tests launching app after termination does not cause issues with DYLD library injection. */

(same comment for test below)